### ISSUE TITLE
Issues with drc_practice_task_2

### ISSUE THREAD

**Issue by osrf-migration** [osrf/drcsim#458]
**Original report ([archived issue](https://osrf-migration.github.io/drcsim-gh-pages/#!/osrf/drcsim/issues/458)) by GoRobotGo (Bitbucket: [GoRobotGo](https://bitbucket.org/%7B2fd8dac8-286d-461a-b7dd-dba99deea025%7D/)).**

The original report had attachments: [Screenshot from 2014-12-15 20:40:18.png](https://osrf-migration.github.io/drcsim-gh-pages/data/repositories/osrf/drcsim/issues/458/attachments/Screenshot_from_2014-12-15_204018.png)

----------------------------------------
drc_practice_task_2 has a few issues with the latest pre-release (4.2.0)
1.  The z-blocks were only 1 high at the trials
2.  The flat steps are all flat in the simulator (see attached image)
3.  The tilted steps didn't have flat steps and went up higher (see attached image)

**Comment by osrf-migration** [comment]
**Original comment by Nate Koenig (Bitbucket: [Nathan Koenig](https://bitbucket.org/%7Bc862cdd9-fcc8-4419-9fc7-e20db14b8fcb%7D/)).**

----------------------------------------
Thanks for the info. I'll take a look at it.

**Comment by osrf-migration** [comment]
**Original comment by Nate Koenig (Bitbucket: [Nathan Koenig](https://bitbucket.org/%7Bc862cdd9-fcc8-4419-9fc7-e20db14b8fcb%7D/)).**

----------------------------------------
Quick fix, in `drc_practice_task_2.world`, add a `<static>true</static>` inside the `<include>` that looks like:


```
#!xml

   <include>
    <name>level_block_0</name>
    <uri>model://block_level_steps</uri>
    <pose>0.351 -0.949 0 0 0 0</pose>
   </include>
```

**Comment by osrf-migration** [comment]
**Original comment by Nate Koenig (Bitbucket: [Nathan Koenig](https://bitbucket.org/%7Bc862cdd9-fcc8-4419-9fc7-e20db14b8fcb%7D/)).**

----------------------------------------
Fix in [pull request #507](https://osrf-migration.github.io/drcsim-gh-pages/#!/osrf/drcsim/pull-requests/507)

**Comment by osrf-migration** [comment]
**Original comment by Ian Chen (Bitbucket: [Ian Chen](https://bitbucket.org/%7Beaa6fca5-6deb-43f6-907f-971c144735dd%7D/), GitHub: [iche033](https://github.com/iche033)).**

----------------------------------------

* changed state from "new" to "resolved"

pull request merged
